content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -37,7 +47,65 @@ In the `generator` block of your `schema.prisma` file, add the following:
 binaryTargets = ["native", "linux-arm64-openssl-1.0.x"]
 ```
 
-## Loading environment variables via a <inlinecode>.env</inlinecode> file
+### Prisma CLI binary targets
+
+While we do not recommend running migrations in an AWS Lambda context, some applications will require it. In these cases, you can use the [PRISMA_CLI_BINARY_TARGETS](https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_cli_binary_targets) environment variable to make sure that `prisma migrate` has access to the correct schema engine.
```

Comment:
What does "AWS Lambda context" refer to here? Is there different/ better terminology we could use here? (It could be my inexperience with AWS 😄 )

Reply:
You're right. Also included some feedback from @janpio 
```suggestion
While we do not recommend running migrations within AWS Lambda, some applications will require it. In these cases, you can use the [PRISMA_CLI_BINARY_TARGETS](https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_cli_binary_targets) environment variable to make sure that Prisma CLI commands, including `prisma migrate`,  have access to the correct schema engine.
```